cds/modules/deposit/static/templates/cds_deposit/types/video/actions.html

```diff
@@ -24,6 +24,16 @@
       <button class="btn btn-primary" ng-click="$ctrl.showCannotEditVideoDialog=false">Close</button>
     </p>
   </modal-dialog>
+  <modal-dialog show="form.infoRNShow" dialog-title="Reserve Report Number" >
+    This action can't be undone.<br>
+    Are you sure you want to reserve a report number?<br>
+    <hr class="mt-30 mb-10" />
+    <p class="pull-right">
+      <button class="btn btn-default" ng-click="$parent.hideModal()">No</button>
+      <button class="btn btn-success" ng-click="$ctrl.actionHandler('RESERVE_REPORT_NUMBER'); $ctrl.cdsDepositCtrl.changeShowAll(true); $parent.hideModal()">Yes</button>
```

Comment:
I would propose to put these 3 click handlers in one method in the controller, but... I don't think it will be better actually :)

Reply:
I wouldn't mind doing it if I knew that I wouldn't break ten different things in the process! 😨